CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## temp
+
+a bunch of nonsense
```

Comment:
What is the purpose of this?

Reply:
ah this was me testing to make sure incremental release notes could be generated from the changelog. I'll remove